### PR TITLE
Contribution to FieldLabComparison

### PR DESCRIPTION
**Pull request by aortiz24**
I have added links for MothuR. I have included the MothuR dependencies.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+ 
 # FieldLabComparison
 
 Scripts for metagenomic analysis of bacterial 16S sequences from ants.
+
+Methods for processing field and lab sample data for comparison and phylogeny building will be performed in MothuR. The program is a free [download] (https://mothur.org/wiki/Download_mothur) and a [tutorial] (https://mothur.org/wiki/454_SOP) is also available.
+
+**Dependencies:**
+* uchime
```
Comment (k8hertweck) [inline README.md]: Could you include URLs to the dependencies?

**Comment by aortiz24** [comment]
I have added the URLs to the MothuR dependencies.

**Comment by k8hertweck** [comment]
Line one has an errant bracket in it, could you remove that?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+]
```
Comment (k8hertweck) [inline README.md]: Could you remove this bracket? I think it's a typo.

File: mothur_otu.bat [hunk 3]
```diff
@@ -1,4 +1,4 @@
-## Microbiome Ant Field vs Lab Analysis OTUs using mothur
+system(cp ~/Desktop/SRP018246/mergegroups SRP018246.groups)## Microbiome Ant Field vs Lab Analysis OTUs using mothur
```
Comment (k8hertweck) [inline mothur_otu.bat]: I think this system command was entered here by mistake. Please remove.

File: mothur_otu.bat [hunk 4]
```diff
@@ -47,7 +47,7 @@ fastq.info(fastq=~/Desktop/SRP018246/SRR655364.fastq)
...
 
 # use mothur system command to copy file to new name
-
+system(cp ~/Desktop/SRP018246/mergegroups SRP018246.groups)
```
Comment (k8hertweck) [inline mothur_otu.bat]: I think you want the groups file to be copied to your project directory (~/Desktop/SRP018246), but right now it's being copied to your home directory (when no path is specified). We can discuss the organization of the project directory at our next meeting.

File: mothur_otu.bat [hunk 5]
```diff
@@ -71,3 +71,20 @@ unique.seqs(fasta=~/Desktop/SRP018246/~/Desktop/SRP018246.fasta)
 # inspect unique sequences
 # input: unique.fasta file, output: unique.summary file
 summary.seqs(fasta=~/Desktop/SRP018246/~/Desktop/SRP018246.unique.fasta, name=~/Desktop/SRP018246/~/Desktop/SRP018246.names)
+
+#download version 128 full length silva reference sequences and seed silva reference file and rename files
+system(cp Silva.nr_128 FullLengthSilva.fasta)
```
Comment (k8hertweck) [inline mothur_otu.bat]: Adding note here that we'll discuss project directory organization in regards to the Silva files at our next meeting.

File: mothur_otu.bat [hunk 6]
```diff
@@ -71,3 +71,20 @@ unique.seqs(fasta=~/Desktop/SRP018246/~/Desktop/SRP018246.fasta)
 # inspect unique sequences
 # input: unique.fasta file, output: unique.summary file
 summary.seqs(fasta=~/Desktop/SRP018246/~/Desktop/SRP018246.unique.fasta, name=~/Desktop/SRP018246/~/Desktop/SRP018246.names)
+
+#download version 128 full length silva reference sequences and seed silva reference file and rename files
+system(cp Silva.nr_128 FullLengthSilva.fasta)
+system(cp Silva.seed_128 SilvaSeed.fasta)
+
+#trim sequences
```
Comment (k8hertweck) [inline mothur_otu.bat]: Let's go ahead and remove this last section (everything from copying the Silva library above, to the end of this current script) and create a new script to correctly format the Silva reference.

File: mothur_otu.bat (hunk 4 above)
Comment (aortiz24) [inline mothur_otu.bat]: Yes, I do want it in my project directory. I have made the necessary change here.

File: mothur_otu.bat (hunk 5 above)
Comment (aortiz24) [inline mothur_otu.bat]: Done!

File: mothur_otu.bat (hunk 3 above)
Comment (aortiz24) [inline mothur_otu.bat]: Removed

File: mothur_otu.bat (hunk 6 above)
Comment (aortiz24) [inline mothur_otu.bat]: Done!

File: README.md (hunk 2 above)
Comment (aortiz24) [inline README.md]: It was a typo.

**Comment by k8hertweck** [comment]
Looks good, thanks!

**Comment by k8hertweck** [comment]
Thanks, Andrea! I've merged your PR.

**Pull request merged**